renovate/dependencyDashboard.js

```diff
@@ -0,0 +1,30 @@
+export class Dependency {
+  constructor(name, version) {
+    this.name = name;
+    this.version = version;
+  }
+}
+
+export const getDependenciesForRepo = ({ octokit }, repo) => {
+  return octokit
+    .request(repo.issuesUrl)
+    .then((response) => {
+      const dependencies = response
```

Comment:
I think this would be easier to test if we only tested the functionality in this `then` block rather than the `octokit` code as well. We don't own the `octokit` code.

Reply:
I've updated the tests.